tool/src/kotlin/formatter.rs

```diff
@@ -40,7 +40,7 @@ impl<'tcx> KotlinFormatter<'tcx> {
 
     pub fn fmt_primitive_to_native_conversion(&self, name: &str, prim: PrimitiveType) -> String {
         match prim {
-            PrimitiveType::Bool => format!("{name}.toByte()"),
+            // PrimitiveType::Bool => format!("{name}.toByte()"),
```

Comment:
?

Reply:
woops, forgot to delete this commented out line -- this case was always wrong, there is no `toByte()` method on `Boolean` in Kotlin. Also, `Boolean` passes fine across the FFI boundary.